libs/dmlf/include/dmlf/update.hpp

```diff
@@ -22,6 +22,7 @@
 #include "crypto/hash.hpp"
 #include "crypto/sha256.hpp"
 #include "dmlf/update_interface.hpp"
+#include "ml/dataloaders/word2vec_loaders/vocab.hpp"
```

Comment:
i think this is fine for now, but at some point in the future we'll probably rework this such that we have an abstract update and inherited updates; in this case a W2VUpdate. Then the serialisers will have the same inheritance structure.